docs/features/autoupdater.rst

```diff
@@ -15,8 +15,8 @@ ecdsautils) can by found in the `contrib` directory. When creating the manifest,
 be set on the command line, or it can be taken from the ``site.mk``.
 
 The priority defines the maximum number of days that may pass between releasing an update and installation
-of the images. The update probability with start at 0 after the release time mentioned in the manifest
-and then slowly rise to 1 after the number of days given by the priority has passed.
+of the images. The update probability will start at 0 after the release time mentioned in the manifest
+and then slowly rise to 1 up to the point when the number of days given by the priority have passed.
```

Comment:
I think "has" is correct, as the subject is "the number of days", which is singular.